src/views/TipsList.vue

```diff
@@ -73,10 +77,12 @@ import TipRecord from '../components/tipRecords/TipRecord.vue';
 import SendTip from '../components/layout/SendTip.vue';
 import LeftSection from '../components/layout/LeftSection.vue';
 import RightSection from '../components/layout/RightSection.vue';
+import MobileNavigation from '../components/layout/MobileNavigation.vue';
 import { EventBus } from '../utils/eventBus';
 import Loading from '../components/Loading.vue';
 import Onboarding from '../components/onboarding/Wizard.vue';
 import { MIDDLEWARE_URL } from '../config/constants';
+import iconEraser from '../assets/iconEraser.svg';
```

Comment:
you missed one

Reply:
Thanks. Also I will restore the back arrow for other resolutions than mobile in the next commit